swarm/network/stream/common_test.go

```diff
@@ -50,14 +50,20 @@ var (
 	peerCount  func(discover.NodeID) int
 	adapter    = flag.String("adapter", "sim", "type of simulation: sim|socket|exec|docker")
 	loglevel   = flag.Int("loglevel", 2, "verbosity of logs")
+	nodes      = flag.Int("nodes", 0, "number of nodes")
+	chunks     = flag.Int("chunks", 0, "number of chunks")
 )
 
 var (
-	defaultSkipCheck bool
-	waitPeerErrC     chan error
-	chunkSize        = 4096
-	registries       map[discover.NodeID]*TestRegistry
-	createStoreFunc  func(id discover.NodeID, addr *network.BzzAddr) (storage.ChunkStore, error)
+	defaultSkipCheck  bool
+	defaultDoRetrieve bool
+	waitPeerErrC      chan error
+	chunkSize         = 4096
+	registries        map[discover.NodeID]*TestRegistry
+	createStoreFunc   func(id discover.NodeID, addr *network.BzzAddr) (storage.ChunkStore, error)
+	getRetrieveFunc   = defaultRetrieveFunc
+	doRetrieve        = defaultDoRetrieve
```

Comment:
this is not used actually, so remove it and remove from defer or 
use it and skip the RPC subscribe call

Reply:
removed